src/Nancy.Authentication.Basic/BasicAuthentication.cs

```diff
@@ -2,6 +2,7 @@
 {
     using System;
     using System.Text;
+

```

Comment:
PR is fucked. Skip.

On 10 November 2015 at 07:58, Kristian Hellang <notifications@github.com>
wrote:

> In src/Nancy.Authentication.Basic/BasicAuthentication.cs
> <https://github.com/NancyFx/Nancy/pull/2098#discussion_r44375165>:
>
> > @@ -2,6 +2,7 @@
> >  {
> >      using System;
> >      using System.Text;
> > +
>
> What's up with all these?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NancyFx/Nancy/pull/2098/files#r44375165>.
>